code/drasil-printers/Language/Drasil/TeX/Print.hs

```diff
@@ -321,20 +326,13 @@ pUnit (UDiv n d) = toMath $
 
 makeDefn :: PrintingInformation -> [(String,[LayoutObj])] -> D -> D
 makeDefn _  [] _ = error "Empty definition"
-makeDefn sm ps l = beginDefn %% makeDefTable sm ps l %% endDefn
-
-beginDefn :: D
-beginDefn = newline
-  %% pure (text "\\noindent \\begin{minipage}{\\textwidth}")
-
-endDefn :: D
-endDefn = pure (text "\\end{minipage}")
+makeDefn sm ps l = newline %% command0 "noindent" %% mkEnvArgs "minipage" "\\textwidth" (makeDefTable sm ps l)
```

Comment:
I think I had asked for the `newline` here (or the `%%` after noindent) because of weird indenting inside the minipage environment; this doesn't seem to be the case anymore. So as long as what's use doesn't produce indent, there is no need for a newline after the noindent.

Reply:
Fixed in #1721- should I do something in this PR about it as well?

Reply:
No. Though there might be interesting merge conflicts arising... we'll see.